app/src/main/java/com/ahmettekin/WeatherApp/view/ListFragment.java

```diff
@@ -76,12 +90,14 @@ private void initViews(View view) {
         recyclerViewAdapter = new RecyclerViewAdapter(weatherItems);
         weatherItems = new ArrayList<>();
         fab = view.findViewById(R.id.floatingActionButton);
+        mContext = view.getContext();
+        searchableSpinner = alertDesign.findViewById(R.id.searchableSpinner);
```

Comment:
initViews'ın içerisine alalım :)

Reply:
mContext ve searchableSpinner ' i mı ? :)

Reply:
searchableSpinner'i alsak yeterli bence 👍

Reply:
searchableSpinner initViews in içinde gözüküyor ama :)

Reply:
Yanlış mı bakıyorum :D

Reply:
haklısın ben öyle görmüşüm :D